x/bank/keeper/keeper_test.go

```diff
@@ -93,7 +94,8 @@ func (suite *IntegrationTestSuite) TestSupply() {
 	totalSupply := sdk.NewCoins(sdk.NewCoin(sdk.DefaultBondDenom, initTokens))
 	suite.NoError(app.BankKeeper.MintCoins(ctx, minttypes.ModuleName, totalSupply))
 
-	total := app.BankKeeper.GetTotalSupply(ctx)
+	total, _, err := app.BankKeeper.GetPaginatedTotalSupply(ctx, &query.PageRequest{})
```

Comment:
Can `GetPaginatedTotalSupply` be private? 

Here, we can still keep `GetTotalSupply`